integration/test_auth.py

```diff
@@ -17,6 +17,7 @@
 import datetime
 import random
 import time
+from typing import List
```

Comment:
I used a few type annotations in the test. They're 3.5, so I *think* they're fair game? I've not tried to use them throughout or anything, and I've avoided the non-test files completely. (Basically, I just added a few so that I could get tab completion. :)